openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -15,6 +15,18 @@ function update_len() {
     $('#excerpts-excerpt-len').html(2000 - len).css('color', color);
 }
 
+export function initEditRow(){
+    document.querySelector('#add_row_button').addEventListener('click', ()=>add_row('website'));
+}
+
+function add_row(name) {
```

Comment:
Could you add a docblock to this function to explain what it does for the next person?

/**
 * @param string name - when prefixed with clone_ should match an element identifier in the page. e.g.  if name would refer to clone_website